website/docs/en/config/entry.mdx

```diff
@@ -7,27 +7,7 @@ import { ApiMeta, Stability } from '../../../components/ApiMeta';
 
 The `entry` configuration is used to set the entry module for Rspack builds.
 
-- **Type:**
-
-  ```ts
-  type EntryDescription = {
-    import: EntryItem;
-    runtime?: EntryRuntime;
-    chunkLoading?: ChunkLoading;
-    asyncChunks?: boolean;
-    publicPath?: PublicPath;
-    baseUri?: string;
-    filename?: EntryFilename;
-  };
-
-  type EntryObject = {
-    [k: string]: EntryItem | EntryDescription;
-  };
-
-  type Config = {
-    entry: EntryItem | EntryObject;
-  };
-  ```
+- **Type:** `string | string[] | Record<string, string | string[] | EntryDescription> | Function`
```

Comment:
Function?

Reply:
It supports (entrychunkname: string) => Record<string, string | string[] | EntryDescription>, I see in other docs use Function to specify such scerios.

Reply:
Get, we can add the function usage to current document.

Reply:
![image](https://github.com/web-infra-dev/rspack/assets/50201324/020e9aba-4b62-433f-833e-66b30913ae18)
It already exists and this part seems to be copied from webpack.

Reply:
get